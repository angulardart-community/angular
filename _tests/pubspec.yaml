name: _tests
description: Tests for AngularDart.
publish_to: none

environment:
  sdk: ">=3.3.0 <4.0.0"

dependencies:
  build: ^2.4.1
  build_resolvers: ^2.4.1
  build_test: ^2.2.1
  collection: ^1.18.0
  glob: ^2.1.2  
  logging: ^1.2.0
  ngcompiler: ^3.0.0-dev.2
  ngdart: ^8.0.0-dev.3
  ngtest: ^5.0.0-dev.2
  source_gen: ^1.4.0
  test: ">=1.24.0 <1.25.0"
  web: ^0.5.1

dev_dependencies:
  analyzer: ^6.0.0
  build_runner: ^2.4.6
  build_web_compilers: ^4.0.4
  csslib: ^1.0.0
  lints: ^2.1.0
  meta: ^1.11.0
  mockito: ^5.4.3
  package_config: ^2.1.0
  source_span: ^1.10.0
  term_glyph: ^1.2.1
