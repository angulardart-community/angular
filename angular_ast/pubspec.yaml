name: angular_ast
repository: https://github.com/angulardart/angular
description: Parser and utilities for AngularDart templates
version: 2.1.1

environment:
  sdk: '>=2.17.0 <3.0.0'

dependencies:
  analyzer: ^4.0.0
  charcode: ^1.1.0
  collection: ^1.14.5
  meta: ^1.1.4
  source_span: ^1.4.0
  string_scanner: ^1.0.1

dev_dependencies:
  build_runner: ^2.0.0
  build_test: ^2.0.0
  path: ^1.6.1
  lints: ^2.0.0
  test: ^1.3.0
