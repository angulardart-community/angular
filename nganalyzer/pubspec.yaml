name: nganalyzer
description: Analyzer plugin for AngularDart.
version: 8.0.0-dev.1
repository: https://github.com/angulardart-community/angular

environment:
  sdk: '>=2.18.0 <3.0.0'

dependencies:
  analyzer: ^5.13.0
  analyzer_plugin: ^0.11.2
  ngast: ^3.0.0-dev.0

dev_dependencies:
  build_runner: ^2.3.3
  build_version: ^2.1.1
  lints: ^2.0.0
  test: ^1.21.0
