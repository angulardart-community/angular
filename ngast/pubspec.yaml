name: ngast
description: Parser and utilities for AngularDart templates
version: 3.0.0-dev.2
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.3.0 <4.0.0"

dependencies:
  charcode: ^1.3.1
  collection: ^1.18.0
  meta: ^1.11.0
  source_span: ^1.10.0
  string_scanner: ^1.2.0

dev_dependencies:
  lints: ^2.1.0
  path: ^1.8.3
  test: ">=1.24.0 <1.25.0"
