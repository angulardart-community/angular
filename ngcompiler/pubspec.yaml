name: ngcompiler
description: >
  The AngularDart compiler.
  It compiles your component files written in Dart into templates so that you
  don't have to worry about all the complicated hassle.
version: 3.0.0-dev.3
repository: https://github.com/angulardart-community/angular

environment:
  sdk: ">=3.3.0 <4.0.0"

dependencies:
  analyzer: ^6.0.0
  args: ^2.4.2
  build: ^2.4.1
  code_builder: ^4.7.0
  collection: ^1.18.0
  csslib: ^1.0.0
  dart_style: ^2.3.2
  logging: ^1.2.0
  meta: ^1.11.0
  ngast: ^3.0.0-dev.2
  ngdart: ^8.0.0-dev.4
  path: ^1.8.3
  source_gen: ^1.4.0
  source_span: ^1.10.0

dev_dependencies:
  build_test: ^2.2.1
  lints: ^4.0.0
  package_config: ^2.1.0
  test: ^1.25.7
