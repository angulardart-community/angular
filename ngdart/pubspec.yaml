name: ngdart
version: 7.1.1
description: >-
  Fast and productive web framework by Google.
  Separate, but similar to the JavaScript framework with a similar name.
repository: https://github.com/angulardart-community/angular
environment:
  sdk: '>=2.18.0 <3.0.0'

dependencies:
  # Runtime (or runtime *and* the compiler).
  built_collection: ^5.1.1
  built_value: ^8.4.2
  collection: ^1.17.0
  intl: ^0.18.0
  js: ^0.6.5
  logging: ^1.1.0
  meta: ^1.8.0
  stack_trace: ^1.11.0
  stream_transform: ^2.1.0

  # Allowing a tiny bit of wiggle here to allow publishing minor updates
  ngast: ^2.1.4
  ngcompiler: ^2.1.4-dev
  ##########################################################################

  build_config: ^1.1.1
  build: ^2.3.1
  code_builder: ^4.4.0
  csslib: ^0.17.2
  path: ^1.8.3
  quiver: ^3.2.1
  source_gen: ^1.2.6
  source_span: ^1.9.1

dev_dependencies:
  # Compiler. Eventually we want to move these to angular_compiler.
  analyzer: ^5.3.1
  build_runner: ^2.3.3
  lints: ^2.0.1
