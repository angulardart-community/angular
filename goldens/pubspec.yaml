name: goldens
description: Goldens for AngularDart.
publish_to: none

environment:
  sdk: ">=3.3.0 <4.0.0"

dependencies:
  meta: ^1.11.0
  ngdart: ^8.0.0-dev.4
  web: ^0.5.1

dev_dependencies:
  build_runner: ^2.4.6
  build_web_compilers: ^4.0.4
  dart_style: ^2.3.3
  glob: ^2.1.2
  lints: ^3.0.0
  path: ^1.8.3
